signing_service/signing_service/utils.py

```diff
@@ -110,7 +113,43 @@ def __call__(  # type: ignore
         setattr(namespace, self.dest, self.const)
 
 
-class Notifier(object):
+class Notifier():
 
-    def send(self) -> None:
-        pass
+    def __init__(
+        self,
+        from_email_address: str,
+        from_email_password: str,
+        to_email_addresses: list,
+    ) -> None:
+        self.from_email_address = from_email_address
+        self.from_email_password = from_email_password
+        self.to_email_addresses = to_email_addresses
+
+    def send(
+        self,
+        message: str,
+    ) -> None:
+        message_to_send = MIMEMultipart()
+        message_to_send['From'] = "Signing Service Notifier"
+        message_to_send['To'] = ','.join(self.to_email_addresses)
+        message_to_send['Subject'] = "Signing Service notification"
+        message_to_send.attach(MIMEText(message))
+        if self.from_email_address is not None and self.from_email_password is not None:
+            self.send_email(message_to_send)
+        print(message_to_send)
```

Comment:
This must be removed: in `EmailNotifier` we just send the message, in `ConsoleNotifier` we log it through a logger

Reply:
Done